### DISCUSSION TITLE
Metaballs + Materials

### DISCUSSION THREAD

**Discussion by Marcus-Richmond** [nortikin/sverchok#4584] [Q&A] [answered]
## Problem statement

I'm trying to control the material of metaball(s) with the vertex color node, but am getting the error:  

![image](https://user-images.githubusercontent.com/59568811/180109245-6be0af90-db5b-4b9c-a2b1-ee7d6e7a7145.png)

Is this an actual error or a limitation of metaballs? Is there any way to control the material of a metaball with Sverchok?

Thanks,

**Comment by Durman** [comment, marked as answer]
Blender does not support vertex groups for meta balls (at least there is no UI for that). Assigning material should work with Assign Materials List node. It works but currently it throughs an error (with metaballs).

![image](https://user-images.githubusercontent.com/28003269/180129728-0ff4df16-cbde-4d28-b7d1-5021d2d35698.png)
